src/lib.rs

```diff
@@ -341,30 +341,53 @@ impl Database {
             #[cfg(feature = "fontconfig")]
             {
                 let mut fontconfig = fontconfig_parser::FontConfig::default();
-                match fontconfig.merge_config("/etc/fonts/fonts.conf") {
-                    Ok(_) => {
-                        for dir in fontconfig.dirs {
-                            let path = if dir.path.starts_with("~") {
-                                if let Ok(ref home) = std::env::var("HOME") {
-                                    let home_path = std::path::Path::new(home);
-                                    home_path.join(dir.path.strip_prefix("~").unwrap())
-                                } else {
-                                    continue;
-                                }
-                            } else {
-                                dir.path
-                            };
-                            self.load_fonts_dir(path);
-                        }
 
-                        // Yes, stop here. No need to load fonts from hardcoded paths.
-                        return;
+                let mut config_files = vec![
+                    std::path::PathBuf::from("/etc/fonts/fonts.conf"),
+                    std::path::PathBuf::from("/etc/fonts/fonts.dtd"),
+                    std::path::PathBuf::from("/etc/fonts/conf.d"),
```

Comment:
Do we need this one? Can you post its content?

Reply:
Do you mean this /etc/fonts/conf.d? Its mentioned in their spec here https://www.freedesktop.org/software/fontconfig/fontconfig-user.html

Reply:
Oh, you are right. It seems we don't need this, since it's referenced in an <include> directive from master fonts.conf file. Which is already handled by `fontconfig-parser`

Reply:
If I read this correctly, `conf.d` is actually a directory and should be included in `fonts.conf` anyway. Not sure if `fontconfig-parser` supports the `include` detective.

I guess we can leave it for now.

Reply:
Yep, good!